internal/tailscale/tailscale.go

```
@@ -28,76 +27,54 @@ const (
 		return nil, fmt.Errorf("failed to get Tailscale status: %w", err)
 	}
-	
+
 	if status.Self == nil || len(status.Self.TailscaleIPs) == 0 {
 		return nil, fmt.Errorf("no Tailscale IPs available")
 	}
-	
+
 	// Listen on the first Tailscale IP
 	addr := fmt.Sprintf("%s:%d", status.Self.TailscaleIPs[0], port)
 	return net.Listen("tcp", addr)
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)